src/main/java/homework_7/Main.java

```diff
@@ -0,0 +1,24 @@
+package homework_7;
+
+public class Main {
```

Comment:
Nice, great example with type conversion!

Reply:
Are there any issues in homework 7? Label has not been tagged

Reply:
No issues, added